ctrl.c

```diff
@@ -227,6 +241,7 @@ static int pv_ctrl_process_signal(int req_fd, size_t content_length,
 	pv_log(DEBUG, "reading and parsing signal...");
 
 	if (content_length >= HTTP_REQ_BUFFER_SIZE) {
+		pv_ctrl_flush_req(req_fd, content_length);
```

Comment:
i find the name of the function confusing as it is not flushing, but rather consuming ... so maybe lets rename it so we dont get confused in future.